<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Music</class>
 <widget class="QMainWindow" name="Music">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Affichage-Options</string>
    </property>
    <widget class="QComboBox" name="piste">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>62</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Piste</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>110</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tempo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>150</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dynamique</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tempo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dyna">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>150</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="translate">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Translation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>48-68</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sauver_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Sauvegarder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="charger_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Charger</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Partition</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>121</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>Actions-MIDI</string>
    </property>
    <widget class="QPushButton" name="sauver">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Sauvegarder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="charger">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Charger</string>
     </property>
    </widget>
    <widget class="QPushButton" name="convertir">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Convertir</string>
     </property>
    </widget>
    <widget class="QLabel" name="aide">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>92</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="aide_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>373</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="insert">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ins√©rer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="retirer">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Retirer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Vider</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="seq">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>391</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>MIDI</string>
    </property>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionCharger"/>
    <addaction name="separator"/>
    <addaction name="actionConvertir"/>
   </widget>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSauvegarder">
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionCharger">
   <property name="text">
    <string>Charger</string>
   </property>
  </action>
  <action name="actionConvertir">
   <property name="text">
    <string>Convertir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
